app/src/main/java/com/rockthevote/grommet/ui/eventFlow/SessionProgressDialogFragment.java

```diff
@@ -50,6 +55,12 @@ static SessionProgressDialogFragment newInstance() {
     public void onCreate(@Nullable Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         Injector.obtain(getContext()).inject(this);
+
+        viewModel = new ViewModelProvider(this,
```

Comment:
You should use `getViewLifecycleOwner()` instead of `this` otherwise there's a potential for this to get out of sync in the lifecycle or interact with the view before its ready.

Also for fragments, you're generally going to want to do this in `onViewCreated` or (now deprecated) `onActivityCreated`